### PR TITLE
drop vsphere-related non-migrated jobs

### PR DESCRIPTION
**Pull request by ameukam**
Drop specific prowjobs depending on non-migrated credentials

### CODE REVIEW

**Comment by ameukam** [comment]
Ref:
  - https://github.com/kubernetes/test-infra/issues/33226
  
  /assign @BenTheElder @michelle192837 
  /hold

**Review by michelle192837 — APPROVED** [review]
/hold

Unhold at will!

(There's some azure jobs dropped in job-migration-todo.md, but I believe that's legitimate given some azure job migration from a separate PR that merged about 13 hours ago).

File: config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-main-periodics.yaml [hunk 1]
```diff
@@ -484,42 +194,3 @@ periodics:
     testgrid-num-failures-to-alert: "4"
     description: Shows test coverage for CAPV
 
-- name: periodic-cluster-api-provider-vsphere-janitor
```
Comment (BenTheElder) [inline config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-main-periodics.yaml:487]: we should probably do this in two phases, so the janitor gets a chance to run after the jobs are turned down?

**Comment by BenTheElder** [comment]
/lgtm-cancel
(to prevent pointless /retest)

File: config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-main-periodics.yaml (hunk 1 above)
Comment (ameukam) [reply config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-main-periodics.yaml]: Agreed. restored.

**Comment by BenTheElder** [comment]
/lgtm
/approve

Feels bad but we have communicated this pretty heavily and need to keep making progress towards overall migration.

**Comment by aojea** [comment]
/unhold cancel

this is a git repo, people willing to recover them just need to revert this

**Comment by ameukam** [comment]
FYI @sbueringer @chrischdi @fabriziopandini

**Review by chrischdi — COMMENTED** [review]
@sbueringer I'm going to cross-check all

**Comment by sbueringer** [comment]
Thx!

File: config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-release-1-11-presubmits.yaml [hunk 2]
```diff
@@ -226,394 +140,3 @@ presubmits:
...
-          requests:
-            cpu: "4000m"
-            memory: "6Gi"
-    annotations:
-      testgrid-dashboards: vmware-cluster-api-provider-vsphere, sig-cluster-lifecycle-cluster-api-provider-vsphere
-      testgrid-tab-name: pr-e2e-supervisor-release-1-11
-      description: Runs all e2e tests
-
-  - name: pull-cluster-api-provider-vsphere-e2e-vcsim-supervisor-release-1-11
-    cluster: eks-prow-build-cluster
```
[the remark this answers is not in the document]
Comment (BenTheElder) [reply config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-release-1-11-presubmits.yaml:448]: thanks for the fix in https://github.com/kubernetes/test-infra/pull/33249, adding a note to the comment in #33226 as well (clearing house for any removed jobs to track back to the PRs)

**Pull request merged**